net/pfSense-pkg-pfBlockerNG-devel/files/usr/local/pkg/pfblockerng/pfblockerng_install.inc

```diff
@@ -169,6 +169,8 @@ unlink_if_exists('/usr/local/bin/php_pfb');
 link('/usr/local/bin/php', '/usr/local/bin/php_pfb');
 unlink_if_exists('/usr/local/sbin/clog_pfb');
 link('/usr/local/sbin/clog', '/usr/local/sbin/clog_pfb');
```

Comment:
This line should be a conditional and only create the link if /usr/local/sbin/clog does exist